charts/stable/xen-orchestra/values.yaml

```
@@ -6,7 +6,7 @@ service:
   main:
     ports:
       main:
-        procotol: HTTPS
+        procotol: https
```

```suggestion
        protocol: https
```
Took me a while to figure ...
It should have errored before if even installed, as caps are not allowed :P